src/main/java/com/sap/cloudfoundry/client/facade/adapters/CloudFoundryClientFactory.java

```diff
@@ -69,6 +70,7 @@ private HttpClient getAdditionalHttpClientConfiguration(HttpClient client) {
         if (getResponseTimeout().isPresent()) {
             clientWithOptions = clientWithOptions.responseTimeout(getResponseTimeout().get());
         }
+        clientWithOptions = clientWithOptions.metrics(true, Function.identity());
```

Comment:
What does function identity do here?

Reply:
Actually nothing as before, the uris will be returned the same way as before, but now the method override "metrics(true)" does not exist anymore